pulumi/src/index.ts

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright 2018, James Nugent.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+ * If a copy of the MPL was not distributed with this file, You can obtain one at
+ * http://mozilla.org/MPL/2.0/.
+ */
+
+import * as aws from "@pulumi/aws";
+import { all, asset, ComponentResource, Output, ResourceOptions } from "@pulumi/pulumi";
+
+/**
+ * The Let's Encrypt ACME v2 Staging URL
+ */
+export const letsEncryptV2StagingUrl = "https://acme-staging-v02.api.letsencrypt.org/directory";
+
+/**
+ * The Let's Encrypt ACME v2 Production URL
+ */
+export const letsEncryptV2ProductionUrl = "https://acme-v02.api.letsencrypt.org/directory";
+
+/**
+ * Tags is a dictionary object representing tags to be applied to an AWS resource.
+ */
+export interface Tags {
```

Comment:
Have we haven't merged your `aws.Tags` change yet? (We should :smile:) Would that let you just use that directly?

Reply:
It _has_ been merged, but isn't part of the `0.14.5` release that is current (https://github.com/pulumi/pulumi-aws/commits/v0.14.5) - so I didn't want to depend on it until I can make this package depend on a stable release. I'll open an issue to fix this up though!